students/DrewSmith/session02/series.py

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env python3
+"""
+Module for generating adding sequence index values.
+"""
+def fibonacci(n):
+    """
+    Returns the nth value in the fibonacci sequence.
+    
+    :param n: the zero based index value in fibonacci sequence to return
+    """
+    first , second, val = 0, 1, 0
+    if n <= 0:
+        return first
+    elif n == 1:
+        return second
+
+    for __ in range(2, n + 1):
+        val = first + second
+        first, second = second, val
+    return val
+
+def lucas(n):
+    """
+    Returns the nth value in the lucas sequence.
+    
+    :param n: the zero based index value in lucas sequence to return
+    """
+    first , second, val = 2, 1, 0
```

Comment:
did you notice that this is exactly the same as fibonacci except this line?

inspiration for the next one :-)